.gitignore

```diff
@@ -5,3 +5,5 @@ target/
 .classpath
 .project
 .settings/
+node_modules
+package-lock.json
```

Comment:
Add the appropriate slashes so that this does not pick other stuff by mistake:
```
/node_modules/
/package-lock.json
```

Reply:
Fixed in https://github.com/sjrd/scala-js-ts-importer/pull/56/commits/d6e7ff0dd73433950622fc24d0b5f1784d6e785f